build/linux/dist/arduino

```diff
@@ -27,7 +27,7 @@ fi
 # Collect options to java in an array, to properly handle whitespace in options
 JAVA_OPTIONS=("-DAPP_DIR=$APPDIR" "-Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel")
 
-if [[ "$@" != *"--upload"* && "$@" != *"--verify"* && "$@" != *"--get-pref"* && "$@" != *"--pref"* && "$@" != *"--install-board"* && "$@" != *"--install-library"* ]] ; then
+if [[ "$@" != *"--upload"* && "$@" != *"--verify"* && "$@" != *"--get-pref"* && "$@" != *"--save-prefs"* "$@" != *"--install-boards"* && "$@" != *"--install-library"* ]] ; then
```

Comment:
Right, I adapted my commit but not the commit-message. I chose to add `--save-prefs` instead of adding both `--board` and `--pref` since they go together when used CLI-only. In fact, if you would be starting the IDE with `--board` (which I guess is possible) the splash will still be there.

Maybe this does not cover all the use-cases, but it covers my use-cases cfr. the man-page:
https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc